powerhome/playbook
# [PLAY-794] Converting User Badge kit to Typescript

**What does this PR do?**
Converting User Badge kit to Typescript.

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-794)

**How to test?**
1. Go to the User Badge kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
